CONTRIBUTING.md

```diff
@@ -1,3 +1,456 @@
-Pull requests and bug reports are welcome!
+# Contributing Guidelines
 
-For larger changes please create an Issue in GitHub first to discuss your proposed changes and possible implications.
+Welcome to Function Clarity. We are excited about the prospect of you joining our community! The Function Clarity communitry abides by the CNCF [code of conduct](https://github.com/cncf/foundation/blob/main/code-of-conduct.md). Here is an excerpt:
```

Comment:
`FunctionClarity` in PascalCase / MixedCaps to be consistent with the other projects in the OpenClarity suite.